prysm.ps1

```diff
@@ -14,12 +13,12 @@ Use this script to download the latest Prysm release binaries.
 Downloaded binaries are saved to .\dist
  
 To specify a specific release version:
-  $env:USE_PRYSM_VERSION=v1.0.0-alpha3
+  `$env:USE_PRYSM_VERSION="v1.0.0-beta.3"
```

Comment:
There is only one back tick here, do you need a closing back tick?

Reply:
The escape character for Powershell is a backtick `, where it for example is `\" in some other programming languages. No closing back tick is necessary.